proxy/src/main/java/org/apache/rocketmq/proxy/processor/DefaultMessagingProcessor.java

```diff
@@ -124,6 +124,12 @@ public static DefaultMessagingProcessor createForClusterMode(RPCHook rpcHook) {
 
     protected void init() {
         this.appendStartAndShutdown(this.serviceManager);
+        this.appendStartAndShutdown(producerProcessor);
```

Comment:
It seems that it's not necessary to start processors because this issue is only related to receiptHandleManager.

Reply:
> It seems that it's not necessary to start processors because this issue is only related to receiptHandleManager.

Indeed, this issue is only related to receiptHandleProcessor. But all these processors extend AbstractStartAndShutdown and there is no caller, which may be error-prone over time?